capstone-backend/src/test/java/com/google/sps/data/ClubTest.java

```diff
@@ -28,18 +40,178 @@ public void setUp() {
   public void testConstructor() {
     Assert.assertEquals(club.getName(), NAME);
     Assert.assertEquals(club.getDescription(), DESCRIPTION);
-    Assert.assertEquals(club.getAnnouncement(), "");
-    Assert.assertEquals(club.getPosts().size(), 0);
+    Assert.assertEquals(club.getAssignments().size(), 0);
     Assert.assertEquals(club.getOwnerID(), OWNER);
     Assert.assertEquals(club.getMemberIDs().size(), 0);
     Assert.assertEquals(club.getInviteIDs().size(), 0);
     Assert.assertEquals(club.getGbookID(), ORIGINAL_BOOK);
   }
 
+  @Test 
+  public void testName() {
+    club.setName(NEW_NAME);
+    Assert.assertEquals(club.getName(), NEW_NAME);
+  }
+
+  @Test 
+  public void testDescription() {
+    club.setDescription(NEW_DESCRIPTION);
+    Assert.assertEquals(club.getDescription(), NEW_DESCRIPTION);
+  }
+
+  @Test 
+  public void testAnnouncement() {
+    club.setAnnouncement(ANNOUNCEMENT);
+    Assert.assertEquals(club.getAnnouncement(), ANNOUNCEMENT);
+  }
+
+  @Test
+  public void testAssignment() {
+    club.addAssignment(ASSIGNMENT_ONE);
+
+    Assert.assertEquals(club.getAssignments().size(), 1);
+
+    List<Assignment> list = new ArrayList<>(club.getAssignments());
+    Assert.assertEquals(list.get(0), ASSIGNMENT_ONE);
+
+    club.addAssignment(ASSIGNMENT_TWO);
+
+    Assert.assertEquals(club.getAssignments().size(), 2);
+    list = new ArrayList<>(club.getAssignments());
+    Assert.assertEquals(list.get(0), ASSIGNMENT_ONE);
+    Assert.assertEquals(list.get(1), ASSIGNMENT_TWO);
+  }
+
+  @Test
+  public void testClearPost() {
+    club.addAssignment(ASSIGNMENT_ONE);
+    club.addAssignment(ASSIGNMENT_TWO);
+
+    Assert.assertEquals(club.getAssignments().size(), 2);
+
+    club.clearAssignments();
+
+    Assert.assertEquals(club.getAssignments().size(), 0);
+  }
+
+  @Test 
+  public void testOwner() {
+    club.setOwnerID(NEW_OWNER);
+    Assert.assertEquals(club.getOwnerID(), NEW_OWNER);
+  }
+
+  @Test
+  public void testInvite() {
+    club.invite(MEMBER_ONE);
+
+    Assert.assertEquals(club.getInviteIDs().size(), 1);
+    List<String> list = club.getInviteIDs();
+    Assert.assertTrue(list.contains(MEMBER_ONE));
+
+    club.invite(MEMBER_TWO);
+
+    Assert.assertEquals(club.getInviteIDs().size(), 2);
```

Comment:
Gitpod is showing a lot of errors. I am guessing this tests aren't done.

Reply:
Looks like last commit didn't push, will push and resolve this now.